webhooks/validating/configuration/src/main.rs

```diff
@@ -677,7 +677,7 @@ mod tests {
     }
 
     #[test]
-    fn test_deserialized_has_extra_array_element() {
+    async fn test_deserialized_has_extra_array_element() {
```

Comment:
Why are all of these changed to `async`? They all look synchronous and are using the `#[test]` macro still.

Reply:
`actix_web` now defines a `test` macro that got pulled here, went to fast during upgrade and thought it was already the case, fixed it now.